enclave_manager/avalon_enclave_manager/sgx_work_order_request.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, enclave_type, work_order, ext_data=""):
 
     # Execute work order in Intel SGX worker enclave
     def execute(self):
-        serialized_byte_array = crypto.string_to_byte_array(self.work_order)
+        serialized_byte_array = crypto.str_to_byte_array(self.work_order)
```

Comment:
you could have kept the function name exactly the same ? Is this the only occurrence of `string_to_byte_array`

Reply:
Ok, will rename to `string_to_byte_array`.This is the only place which is used.

Reply:
Done